libraries/lib-sample-track/Mix.cpp

```diff
@@ -238,7 +240,7 @@ size_t Mixer::Process(const size_t maxToProcess)
 
    Clear();
    // TODO: more-than-two-channels
-   auto maxChannels = mFloatBuffers.Channels();
+   auto maxChannels = std::max(2u, mFloatBuffers.Channels());
```

Comment:
Probably it makes sense to have `maxChannels` as constexpr somewhere, and reuse it in both places?